### PR TITLE
🌐 Add Turkish translation for `docs/tr/docs/tutorial/request-forms.md`

### PR DESCRIPTION
**Pull request by hasansezertasan**
🌐 Add Turkish translation for `docs/tr/docs/tutorial/request-forms.md`

[Original File](https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/tutorial/request-forms.md)

Discussion: #9193

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 174b97276281a8ff34183bfef19306eab0224571 at: https://afc2c505.fastapitiangolo.pages.dev

**Review by OzgunCaglarArslan — APPROVED** [review]

File: docs/tr/docs/tutorial/request-forms.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+=== "Python 3.8+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
+
+    ```Python hl_lines="7"
+    {!> ../../../docs_src/request_forms/tutorial001.py!}
+    ```
+
+Örneğin, OAuth2 spesifikasyonunun kullanılabileceği ("şifre akışı" olarak adlandırılan) yollardan birinde, form alanları olarak <abbr title="Kullanıcı Adı: Username">"username"</abbr> ve <abbr title="Kullanıcı Adı: Username">"password"</abbr> gönderilmesi gerekir.
```
Comment (hasansezertasan) [inline docs/tr/docs/tutorial/request-forms.md:60]:
```suggestion
Örneğin, OAuth2 spesifikasyonunun kullanılabileceği ("şifre akışı" olarak adlandırılan) yollardan birinde, form alanları olarak <abbr title="Kullanıcı Adı: Username">"username"</abbr> ve <abbr title="Şifre: Password">"password"</abbr> gönderilmesi gerekir.
```

File: docs/tr/docs/tutorial/request-forms.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
+
+    ```Python hl_lines="7"
+    {!> ../../../docs_src/request_forms/tutorial001.py!}
+    ```
+
+Örneğin, OAuth2 spesifikasyonunun kullanılabileceği ("şifre akışı" olarak adlandırılan) yollardan birinde, form alanları olarak <abbr title="Kullanıcı Adı: Username">"username"</abbr> ve <abbr title="Şifre: Password">"password"</abbr> gönderilmesi gerekir.
+
+Bu <abbr title="Spesifikasyon: Specification">spesifikasyon</abbr> form alanlarını adlandırırken isimlerinin birbir `username` ve `password` olmasını ve JSON verisi yerine form verisi olarak gönderilmesini gerektirir.
```
Comment (hasansezertasan) [inline docs/tr/docs/tutorial/request-forms.md:62]:
```suggestion
Bu <abbr title="Spesifikasyon: Specification">spesifikasyon</abbr> form alanlarını adlandırırken isimlerinin birebir `username` ve `password` olmasını ve JSON verisi yerine form verisi olarak gönderilmesini gerektirir.
```

File: docs/tr/docs/tutorial/request-forms.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+!!! info "Bilgi"
+    `Form` doğrudan `Body` sınıfını miras alan bir sınıftır.
+
+!!! tip "İpucu"
+    Form gövdelerini tanımlamak için `Form` sınıfını kullanmanız gerekir; çünkü bu olmadan parametreler sorgu parametreleri veya gövde (JSON) parametreleri olarak yorumlanır.
+
+## "Form Alanları" Hakkında
+
+HTML formlarının (`<form></form>`) verileri sunucuya gönderirken JSON'dan farklı olan özel bir kodlama kullanır.
```
Comment (hasansezertasan) [inline docs/tr/docs/tutorial/request-forms.md:74]:
```suggestion
HTML formlarının (`<form></form>`) verileri sunucuya gönderirken JSON'dan farklı özel bir kodlama kullanır.
```

**Comment by hasansezertasan** [comment]
@alejsdev, It's ready 🚀 🚀 🚀

**Comment by tiangolo** [comment]
📝 Docs preview for commit 1d3df063e91410325f6dd003c0e53518a04466e7 at: https://ddb01c83.fastapitiangolo.pages.dev

**Comment by alejsdev** [comment]
Nice! Thanks for the help @hasansezertasan @OzgunCaglarArslan :rocket: :nerd_face:

**Pull request merged**